CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+* client/server SSL settings fail when 'ssl.key.password' is set ([#6763](https://github.com/confluentinc/ksql/pull/6763)) ([3e48540](https://github.com/confluentinc/ksql/commit/3e48540fbe1a02ed02faa667e424b225f9d7ba20))
+* do not log data on JSON deserialization errors (MINOR) ([#6930](https://github.com/confluentinc/ksql/pull/6930)) ([1834da6](https://github.com/confluentinc/ksql/commit/1834da678cf86e7cb333d57a9c97d1884d09d7af))
+* dont use the streamthread to construct the log-metric-and-continue logger ([#6909](https://github.com/confluentinc/ksql/pull/6909)) ([6af3bc6](https://github.com/confluentinc/ksql/commit/6af3bc6fc49de977c5571bb0af5188ac186516e9))
+* ff-4240-upgrade httpclient version ([#6935](https://github.com/confluentinc/ksql/pull/6935)) ([19f4d63](https://github.com/confluentinc/ksql/commit/19f4d632bc4c854904f366b359d4bf70a6f54d6d))
+* fix how the buffer limit check evaluates streams config ([#6876](https://github.com/confluentinc/ksql/pull/6876)) ([ad1cc2a](https://github.com/confluentinc/ksql/commit/ad1cc2aca3b3a51f6f9e3aee86b1f41de449ae15))
+* Fixes auth for forwarded requests for pull queries ([#6895](https://github.com/confluentinc/ksql/pull/6895)) ([c360c9c](https://github.com/confluentinc/ksql/commit/c360c9c389bff435c5727b2a1f0373f112cb0f68))
+* Fixes the build by adding groupMetadata as a faked method of the sandbox ([#6936](https://github.com/confluentinc/ksql/pull/6936)) ([444f4f9](https://github.com/confluentinc/ksql/commit/444f4f94ff937abac867c5f0ce67d66a4e42e4f3))
+* format cast arguments with passed context (6.0.x) ([#7032](https://github.com/confluentinc/ksql/pull/7032)) ([8c0d93d](https://github.com/confluentinc/ksql/commit/8c0d93db4ad94f820639ccb993dbc53a81175e53))
+* make formattimestamp and parsetimestamp default to utc ([#6954](https://github.com/confluentinc/ksql/pull/6954)) ([a5ea98a](https://github.com/confluentinc/ksql/commit/a5ea98affd13b99d17c21832c09394e45b1f86ac))
+* Make Select * avoid code gen for projections ([#6846](https://github.com/confluentinc/ksql/pull/6846)) ([0896b85](https://github.com/confluentinc/ksql/commit/0896b856a86c1f7b212d8b5ebe3f9140f5679cd6))
+* npe when getting topic configs ([#6946](https://github.com/confluentinc/ksql/pull/6946)) ([5e026d4](https://github.com/confluentinc/ksql/commit/5e026d4ad9d02629566593705e79d568296e96bd))
+* remove mutable subscriber field on an endpoint ([#6905](https://github.com/confluentinc/ksql/pull/6905)) ([98c7d73](https://github.com/confluentinc/ksql/commit/98c7d735c2506a0204ecce8d8861f07eb62d968d))
+* Select star table scans need keys in the intermediate row ([#6908](https://github.com/confluentinc/ksql/pull/6908)) ([fcac459](https://github.com/confluentinc/ksql/commit/fcac4595ff88a0b9a4f4b91f53eaa0d4f669aa31))
+* use internal format with multi-col support when needed ([#6933](https://github.com/confluentinc/ksql/pull/6933)) ([4bb15c4](https://github.com/confluentinc/ksql/commit/4bb15c421149686a176ef6bdd0ed1e0962f5305f))

```

Comment:
Remove this, this made it into 0.15 (the bug fix was for a new feature released with 0.15 so it doesn't make sense for it to appear on any changelog). Not sure why it's being shown in this changelog as I can see the commit on the 0.15 branch...

Reply:
Looked into this some more -- I think there's a bug in how the changelog generator script handles cherry-picked commits and that's why this is appearing here even though it shouldn't be (since there's another version of this commit on the 0.15 branch). I'll open a PR to fix the script. 

There are a few more instances of items that shouldn't be in the 0.16 changelog for the same reason. Let me go through and mark them so you can remove them.

Reply:
Here's the fix for the changelog generation script: https://github.com/confluentinc/ksql/pull/7233